packages/flex-plugin-scripts/src/scripts/pre-script-check.ts

```diff
@@ -175,6 +175,23 @@ export const _setPluginDir = (...args: string[]): void => {
   }
 };
 
+/**
+ * Compares the CLI and the plugin for flex-plugin-scripts
+ * @private
+ */
+export const _comparePluginAndCLIVersions = (): void => {
+  const cliPackageJson = readPackageJson(join(getCliPaths().dir, packageJsonName));
+  const pluginPackageJson = readPackageJson(join(getCwd(), packageJsonName));
+  const cliVersion = packageDependencyVersion(cliPackageJson, '@twilio-labs/plugin-flex');
+  const pluginVersion = packageDependencyVersion(pluginPackageJson, 'flex-plugin-scripts');
+
+  if (cliVersion && pluginVersion && semver.clean(cliVersion) !== semver.clean(pluginVersion)) {
```

Comment:
hmm, you might want to use `semver.coerce(...).version` since it looks like `semver.clean` would return `null` for `~1.2.3` and `^1.2.3`, which could be potential versions in package.json

Reply:
or actually, `semver.coerce(null)` would return null, so probably
```
if (semver.coerce(cliVersion) && semver.coerce(pluginVersion) && semver.coerce(cliVersion).version !== semver.coerce(pluginVersion).version) {
```
(can also make those into variables so you're not repeating semver.coerce)